apparelo/apparelo/doctype/item_production_detail/item_production_detail.py

```diff
@@ -578,7 +578,7 @@ def process_based_qty(process,ipd=None,lot=None,qty_based_bom=None):
 	additional_item_list = []
 	for item in items:
 		if item['qty']!=0:
-			final_item_list.append({"item_code": item['item'],"qty":item['qty'] ,"uom": item['uom']})
+			final_item_list.append({"item_code": item['item'],"qty":round(item['qty'],2) ,"uom": item['uom']})
```

Comment:
@karthikeyan5 , Rounding off ourself here, do you think, is good?

Reply:
Nope... Can we see how it is done in ERPNext... For example in production plan... And do it that way... Let's not hard code it to 2 digits...